assets/mercury/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+version: "3.4"
+# we can spin it up
+# so that we add it
+#
+services:
+  db:
+    image: postgres:13.7
+    # Turn off db logs by default
+    # https://postgresqlco.nf/doc/en/param/log_statement/
+    command: ["postgres", "-c", "log_statement=none"]
+    environment:
+      POSTGRES_DB: mercury
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    ports:
+      - 5432:5432
+    healthcheck:
+      test: ["CMD-SHELL", "sh -c 'pg_isready -U postgres -d mercury'"]
+      interval: 5s
+      timeout: 2s
+      retries: 5
+  mq:
+    image: rabbitmq:3.10.20-management
+    hostname: gx-bunny
+    environment:
+      RABBITMQ_DEFAULT_USER: gx_prod
+      RABBITMQ_DEFAULT_PASS: password
+    ports:
+      - 8042:15672
+      - 5672:5672
+  db-provisioner:
+    image: 258143015559.dkr.ecr.us-east-1.amazonaws.com/mercury/provisioner:0.4.1
```

Comment:
Do we need to sync up this provisioner version with mercury?

Reply:
The `provisioner` is responsible for creating the database, setting up users, roles, etc for the mercury postgres instance. We'll need to do this to get the db in a state that mercury can use. It does not need to be synced with with `mercury`.